frontend/src/components/dashboard/DataDashboard.vue

```diff
@@ -8,10 +9,11 @@ const props = defineProps(['dataset', 'view'])
 </script>
 
 <template>
-  <div class="data-dashboard-view">
+  <div class="data-dashboard">
     <Demo v-if="props.dataset == 'demo' && props.view == 'explore'" />
+    <IPEDS_318_45 v-if="props.dataset == 'ipeds-318-45' && props.view == 'explore'" />
```

Comment:
Nit: Vue's convention is to use CamelCase. I know that's not really practical with numbers however. I wonder if `<IPEDS-318-45 />` is better to match standard html tag use of kebab case?

But I don't feel strongly about this. If this is the most readable let's keep it. I just noticed it because the syntax highlighting breaks in GitHub.

Reply:
typescript doesn't like minus signs in code. I just tried and failed to convert- importing a var with minuses broke the compilation.

Reply:
resolving but lemme know if you'd like me to try something else.